apps/site/next.config.mjs

```diff
@@ -1,7 +1,11 @@
 'use strict';
 import createNextIntlPlugin from 'next-intl/plugin';
 
-import { BASE_PATH, ENABLE_STATIC_EXPORT } from './next.constants.mjs';
+import {
+  BASE_PATH,
+  ENABLE_STATIC_EXPORT,
+  OPEN_NEXT_CLOUDFLARE,
```

Comment:
nit: can we move all cloudflare-specific constants to `next.cloudflare.constants.mjs` (so we can do the same with Vercel-ones to `next.vercel.constants.mjs` in a follow-up or in this PR if you wish)